apps/heureka/src/components/common/IssuesCountPerSeverityLevel.tsx

```diff
@@ -4,40 +4,15 @@
  */
 
 import React from "react"
-import {
-  Stack,
-  Badge,
-  Tooltip,
-  TooltipTrigger,
-  TooltipContent,
-  KnownIcons,
-  BadgeVariantType,
-} from "@cloudoperators/juno-ui-components"
+import { Stack } from "@cloudoperators/juno-ui-components"
 import { IssuesCountsWithTotalCountType, IssuesCountsType } from "../Services/useFetchServicesIssuesCounts"
+import { SeverityCount } from "./SeverityCount"
 
 type IssuesCountPerSeverityLevelProps = {
   counts: IssuesCountsWithTotalCountType | IssuesCountsType
-  displayMode?: "all" | "criticalHigh"
 }
 
-type ToolTipBadgeType = {
-  icon?: KnownIcons
-  text: string
-  variant: BadgeVariantType
-  tooltipContent: string
-  className?: string
-}
-
-const ToolTipBadge = ({ icon, text, variant, tooltipContent, className }: ToolTipBadgeType) => (
-  <Tooltip triggerEvent="hover">
-    <TooltipTrigger>
-      <Badge icon={icon} text={text} variant={variant} className={className} />
-    </TooltipTrigger>
-    <TooltipContent>{tooltipContent}</TooltipContent>
-  </Tooltip>
-)
-
-export const IssuesCountPerSeverityLevel = ({ counts, displayMode = "all" }: IssuesCountPerSeverityLevelProps) => {
+export const IssuesCountPerSeverityLevel = ({ counts }: IssuesCountPerSeverityLevelProps) => {
```

Comment:
the component name is not completely correct... 
`IssuesCountPerSeverityLevel`
vs
`IssueCountsPerSeverityLevel`